src/main/resources/application-prod.yaml

```diff
@@ -0,0 +1,3 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://A01DBVL010.adeo.no:5432/familie-ef-sak
```

Comment:
Alle applikasjonene til enslig forsørger eer bygget satt opp meed at application.yml er prod-settings. Alle andre miljøer overstyrer. Ikke legg til en appliaction-prod.

Reply:
Ok. Skal prøve. Slet litt med å overstyre settingen i kode. Men det kan jeg sikkert få hjelp til!

Reply:
Bare spør, så kan vi se på det sammen. Jeg sitter selv ofte for lenge alene med ting som ikke fungerer.

Reply:
Fikk det til. Har fjernet application-prod og lagt tilbake prod-connection-string'en i application.yaml